### PR TITLE
feat(photos page): #861mpxdh2: add sort component, devide page into l…

### PR DESCRIPTION
**Pull request by MadSquirrelF**
…ist, card and page, add fake test data for review

### CODE REVIEW

File: src/pages/Photos/components/PhotosList/PhotoList.scss [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    justify-content: space-between;
+    margin-top: 16px;
+
+    @include desktop {
+      margin-top: 0;
+    }
+  }
+
+  &__container {
+    $this: &;
```
Comment (katariniss) [inline src/pages/Photos/components/PhotosList/PhotoList.scss:32]: why do we need it here? 
as a rule it is supposed to be on the block level

File: src/pages/Photos/components/Sort/Sort.tsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    <div
+      ref={sortRef}
+      className="sort"
+      data-cy="sort"
+    >
+      <div
+        className="sort__label"
+        data-cy="sort-label"
+      >
+        <p className="sort__title">Sort by</p>
```
Comment (katariniss) [inline src/pages/Photos/components/Sort/Sort.tsx:39]: I would opt out for a span here, as P tag is for paragraphs

File: src/pages/Photos/PhotosPageContent.tsx [hunk 3]
```diff
@@ -20,19 +22,18 @@ function PhotosPageContent() {
   return (
     <div>
```
Comment (katariniss) [inline src/pages/Photos/PhotosPageContent.tsx:23]: can we get rid of this container? it looks redundant

File: src/pages/Photos/components/Sort/Sort.scss [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+    margin-right: 8px;
+    font-size: 14px;
+    color: $color-dark-grey;
+  }
+
+  &__select-popup-btn {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin-left: 8px;
```
Comment (katariniss) [inline src/pages/Photos/components/Sort/Sort.scss:31]: I think this margin should be moved to the level of the parent

File: src/pages/Photos/components/PhotoCard/PhotoCard.tsx [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    >
+      <div className="photo-card__image-container">
+        <img
+          src={photoPath}
+          className="photo-card__image"
+          alt="test"
+          draggable={false}
+        />
+      </div>
+      <p
```
Comment (katariniss) [inline src/pages/Photos/components/PhotoCard/PhotoCard.tsx:28]: P tag is not suitable here as well

File: src/pages/Photos/components/PhotosList/PhotoList.scss [hunk 6]
```diff
@@ -0,0 +1,44 @@
+.photo-list {
+  position: relative;
```
Comment (katariniss) [inline src/pages/Photos/components/PhotosList/PhotoList.scss:2]: do we really need position relative here?

File: src/pages/Photos/components/PhotosList/PhotoList.scss [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+  display: flex;
+  flex-direction: column;
+  padding-top: $top-menu-height;
+  min-height: 100vh;
+
+  &__card {
```
Comment (katariniss) [inline src/pages/Photos/components/PhotosList/PhotoList.scss:8]: Please note that the elements should be vertically aligned on the right side
Screenshot 2023-05-03 at 7 28 27 PM

File: src/pages/Photos/components/Sort/Sort.tsx [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+            data-cy="sort-popup"
+          >
+            <ul className="sort__popup-list">
+              {
+                sortList.map((sort) => (
+                  <li
+                    key={sort}
+                    data-cy="sort-popup-item"
+                    className={clsx(`sort__popup-item`, {
+                      'sort__popup-item sort__popup-item--active': photosPageState._sort === sort,
```
Comment (katariniss) [inline src/pages/Photos/components/Sort/Sort.tsx:76]: you don't need to duplicate class **sort__popup-item** for the active state

File: src/pages/Photos/components/Sort/Sort.scss [hunk 9]
```diff
@@ -0,0 +1,121 @@
...
+    @include reset-list;
+
+    width: 100%;
+  }
+
+  &__popup-item {
+    display: flex;
+    align-items: center;
+    justify-content: flex-start;
+    padding: 12px 10px;
```
Comment (katariniss) [inline src/pages/Photos/components/Sort/Sort.scss:99]: please double check the spacings on the popup, as visually they don't match the design

Comment (MadSquirrelF) [reply src/pages/Photos/components/Sort/Sort.scss:99]: I changed the margins, but there is confusion in the design.

Mobile padding 
Jbd_XNzZ6l8
lPi90LNlHbY

Desktop
0tG-YvgE7Uo

VB8j4biHlTM

**Pull request merged**